### ISSUE TITLE
issue with CSSStyleSheet.insertRule()

### ISSUE THREAD

**Issue by stereobooster** [stereobooster/react-snap#76]
Some CSS in JS solutions use [`CSSStyleSheet.insertRule()`](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/insertRule). For example, [emotion](https://emotion.sh/). In this case `<style>` tag seen as empty by browser.

Solution is pretty easy:

```js
Array.from(document.styleSheets).map(sheet => {
  return Array.from(sheet.rules).map(x => x.cssText).join("\n")
}).join("\n")
```

Other ways to insert CSS:

```js
var css = 'h1 { background: red; }',
    head = document.head || document.getElementsByTagName('head')[0],
    style = document.createElement('style');

style.type = 'text/css';
if (style.styleSheet){
  style.styleSheet.cssText = css;
} else {
  style.appendChild(document.createTextNode(css));
}

head.appendChild(style);
```

This is the issue behind glamor problem discussed in https://github.com/errorception/snapshotify/issues/1

Reproducible example is [here](https://github.com/stereobooster/emotion/tree/react-snap)

**Comment by stereobooster** [comment]
@peterbe you might be interested too

**Comment by stereobooster** [comment]
Fixed in v1.5.0
